Makefile

```diff
@@ -136,6 +136,7 @@ run-tests:
 		--examples_uri ${examples_uri} \
 		--examples_ref ${examples_ref} \
 		--namespace_name ${namespace_name} \
+		--load_factor ${load_factor} \
```

Comment:
what is this load factor?

Reply:
See https://issues.redhat.com/browse/KOGITO-1338